docs/python-reference/api.rst

```diff
@@ -118,6 +121,15 @@ Group-by (split-apply-combine)
    GroupByDataset.min
    GroupByDataset.sum
 
+Other
+-----
+
+.. autosummary::
+   :toctree: ../generated
+
+   GroupByDataArray.copy
+   GroupByDataset.copy
+
```

Comment:
Should probably also update "Apart from `mean`, `groupby` currently also supports `sum` and `flatten`" in https://github.com/scipp/scipp/blob/master/docs/user-guide/groupby.ipynb